package.json

```diff
@@ -11,17 +11,23 @@
     "db:seed": "npx knex seed:run",
     "test": "SERVER_WALLET_DB_NAME=server_wallet_test npx jest --runInBand",
     "watch-server": "SERVER_WALLET_DB_NAME=server_wallet_development nodemon --watch 'src/**/*' -e ts,tsx --exec ts-node ./src/app/server.ts",
-    "build": "npx tsc"
+    "build": "npx tsc",
+    "start": "node ./lib/app/server.js",
+    "prestart": "npm run db:rollback && npm run db:migrate && npm run db:seed",
+    "postinstall": "npm run build"
   },
   "author": "",
   "license": "MIT",
   "dependencies": {
+    "dotenv": "^6.2.0",
+    "dotenv-expand": "^4.2.0",
     "fmg-core": "^0.5.6",
     "fmg-nitro-adjudicator": "^0.0.10",
     "knex": "^0.13.0",
     "koa": "^2.7.0",
     "koa-body": "^4.0.8",
     "koa-router": "^7.4.0",
+    "magmo-devtools": "git+https://github.com/magmo/devtools.git#v0.1.8",
```

Comment:
This doesn't seem like it should be a dependency, but let's go with it for now.

Reply:
@andrewgordstewart we are using the dependency here: https://github.com/magmo/node-bot/blob/master/src/wallet/utilities/blockchain.ts#L6. That's why it was moved from `devDependencies` to `dependencies`.

Reply:
Yes, but it's a needless dependency that should probably be removed.